queue_job/tests/common.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from odoo.addons.queue_job.job import Job
+
+
+class JobCounter(object):
+
+    def __init__(self, env):
+        super(JobCounter, self).__init__()
+        self.env = env
+        self.existing = self.search_all()
+
+    def count_all(self):
+        return len(self.search_all())
+
+    def count_created(self):
+        return len(self.search_created())
+
+    def count_existing(self):
+        return len(self.existing)
+
+    def search_created(self):
+        return self.search_all() - self.existing
+
+    def search_all(self):
+        return self.env['queue.job'].search([])
+
+
+class JobMixin(unittest.TestCase):
```

Comment:
the base class here probably is irrelevant

Reply:
I admit I've just cherry-picked (and it works) :smiley:

Reply:
sure it works :) I'm just saying that you have to use this mixin together w/ an odoo test class which already inherits from base unittest class.
Plus, here you need and env to work, which means you cannot really use this class standalone. 
Anyway, is just a detail...

Reply:
So I made 2 suggestions (no need to inherit from `object` in python 3)